FavoritesMicro/Sources/FavoritesMicro/FavoriteNavigationView/FavoriteNavigationViewModel.swift

```diff
@@ -1,19 +1,10 @@
-//
-//  SwiftUIView.swift
-//  
-//
-//  Created by Hai Nguyen on 2/15/24.
-//
-
-import Foundation
-import CommonUI
 import DataModels
-import SwiftUI
+import FeatureView
 
 class FavoriteNavigationViewModel {
     // MARK: - View Builders
     
-    func createMovieDetailsView(movie: Movie) -> MovieDetailsView {
+    func createMovieDetailsView(movie: Movie) -> MovieDetailsView? {
```

Comment:
why is this returning an optional `MovieDetailsView`.  There will always be a returned object